requirements.txt

```diff
@@ -2,4 +2,5 @@ Mako
 PyYAML
 six
 pbr
-defusedxml
+# For OpenNebula provider
+# defusedxml
```

Comment:
Maybe  better to use extras?
https://pip.readthedocs.io/en/1.1/requirements.html#the-requirements-file-format